test/LicenseTests/Helpers/LicenseOptionTestHelper.cs

```diff
@@ -31,17 +31,20 @@ public static string CreateLicenseJsonFile(string directoryPath, string licenseT
         return licenseJsonPath;
     }
 
+
+    static string Normalize(string s) => s.Replace("\r\n", "\n");
+
     internal static void AssertLicenseVerificationException(Exception exception, LicenseHelper.LicenseVerificationErrorBuilder expectedMessageBuilder)
     {
-        Assert.True(exception.Message.Contains(expectedMessageBuilder.ToString()),
```

Comment:
In RavenTestHelper we already have:
- AssertEqualRespectingNewLines
- AssertStartsWithRespectingNewLines

I think what you need to add is AssertContainsRespectingNewLines